core/src/main/java/hudson/model/Queue.java

```diff
@@ -1,9 +1,10 @@
 /*
  * The MIT License
- *
+ * 
```

Comment:
Please revert unrelated changes so that a diff is minimal before submitting a PR. (Makes reviews easier, reduces changes of merge conflicts, avoids noise in `git blame`, etc.)